cli/compose/types/types.go

```diff
@@ -6,7 +6,6 @@ import (
 
 // UnsupportedProperties not yet supported by this implementation of the compose file
 var UnsupportedProperties = []string{
-	"build",
```

Comment:
We need to keep `build` in the list of unsupported properties because it's still not used by the cli right now.

Reply:
@dnephin Okay :+1: